playbooks/common/openshift-cluster/set_node_launch_facts_tasks.yml

```diff
@@ -1,11 +1,13 @@
 ---
-- set_fact: k8s_type="node"
+- set_fact: k8s_type=node
+- set_fact: sub_host_type="{{ type }}"
+- set_fact: number_nodes="{{ count }}"
 
 - name: Generate node instance names(s)
   set_fact:
-    scratch_name: "{{ cluster_id }}-{{ k8s_type }}-{{ '%05x' | format(1048576 | random) }}"
+    scratch_name: "{{ cluster_id }}-{{ sub_host_type }}-{{ '%05x' | format(1048576 | random) }}"
```

Comment:
So the result is whearntest-node-node-a9f59, should we set the sub_host_type to compute on the normal nodes?